### ISSUE TITLE
Creating issues to update alt text for all projects in the _projects collection

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#3866]
### Dependency
- [x] #3998
- [x] #4012
- [x] #3868
- [x] #3869
- [x] #3870
- [x] #3873
- [x] #3874
- [x] #3904
- [x] #3933
- [x] #3948 
- [x] #3950
- [x] #3951
- [x] #3952
- [x] #3953
- [x] #3986 
- [x] #3987
- [x] #3997
- [x] #3999
- [x] #4000
- [x] #4006
- [x] #4007
- [x] #4008
- [x] #4013
- [x] #4014
- [x] #4015
- [x] #4045
- [x] #4046
- [x] #4047
- [x] #4049
- [x] #4044
- [x] #4043
- [x] #4042
- [x] #4041
- [x] #4040
- [x] #4037
- [x] #4036
- [x] #4031
- [x] #4026
- [x] #4023

### Overview
We need to ensure the alt text for project images adhere to Web Content Accessibility Guidelines (WCAG). For this epic, we will write issues to change the alt text for all project files in the `_projects` collection.

### Details
Product Manager identified that the alt text for project images should be changed because they are either inaccurate or too long. According to the WCAG page, alt text should be descriptive but short and succinct. 

### Instructions
- For each project, we will create an issue using the Title Template and Body Template from the Resources section of this issue. 
- `[ PROJECT NAME ]`, `[ PROJECT MD FILE ]`, and `[ PROJECT SLUG ]` will be replaced with the corresponding values for each project. 
- If you start with the [Alt Text Audit - Dev](https://github.com/hackforla/website/issues/new?assignees=&labels=Feature%3A+Accessibility%2C+good+first+issue%2C+ready+for+dev+lead%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+size%3A+1pt&template=alt-text-audit--dev.md&title=%5BINSERT+PAGE+HERE%5D%3A+Update+alt+text+for+%5BINSERT+IMAGE+NAME+HERE%5D+image+to+adhere+to+WCAG) issue template, most of the appropriate labels will be automatically added, but you will have to replace both the title and body of the issue completely with the templates provided below.

### Action Items
- [x] This issue should only be assigned to a lead.
- [x] In the Dependency section of this issue, replace each project name with a link to the newly created issue that addresses changing the project's alt value
- [x] Once all the necessary issues have been created and added to this issue's dependency section, add a `dependency` label and move this issue to icebox column.
- [x] If all the dependency issues are closed, remove the dependency label, move this issue from `ice box` to `In progress` column and close this issue.

### Resources
- [_projects collection folder](https://github.com/hackforla/website/tree/gh-pages/_projects)
- [How to create issues](https://github.com/hackforla/website/wiki/How-to-create-issues) wiki
- [PM Comment that sparked this epic issue](https://github.com/hackforla/website/issues/2613#issuecomment-1401292233)

#### Title Template
```
[ PROJECT NAME ]: Update alt text for the [ PROJECT NAME ] image to adhere to WCAG
```

#### Body Template

````
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As a developer, we need to provide clear and descriptive alt text for the [PROJECT NAME] image so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.
### Action Items
- [ ] Change the alt value within `_projects/[ PROJECT MD FILE ]`:

**From:**
```
alt: [ REPLACE WITH CURRENT ALT VALUE ]
```
**To:**
```
alt: [ PROJECT NAME ]
```
**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
-  [ PROJECT NAME ] Page: https://www.hackforla.org/projects/[ PROJECT SLUG ] 

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE:  `_projects/[ PROJECT MD FILE ]`
- See the source file code on the repo: https://github.com/hackforla/website/blob/gh-pages/_projects/[ PROJECT MD FILE ]
- This issue is part of epic #3866

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
````

**Comment by ExperimentsInHonesty** [comment]
@MattPereira is there a template for the issues that you are making?

### Example
In my example, I didn't make a template in the repo because it was only three issues: https://github.com/hackforla/website/issues/3891

But given that the issues you will be making are many more, I would make a template, that way all the labels are automatically added each time you use it.  But you can see from my template, I put in placeholders [TEXT IN CAPS] wherever I needed to replace something in the new issue.

**Comment by MattPereira** [comment]
**Progress**: Most of the issues are created
**Blockers**: No blockers just a long tedious grind
**Availability**: 6 hours
**ETA**: End of Week

**Comment by chriskong217** [comment]
The project slug should be the actual name of the project, and should not include .md at the end. Putting .md at the end of the link will result in a 404 error. 
In your template above, it has the following formatting:

> ### Resources/Instructions
> -  [ PROJECT NAME ] Page: https://www.hackforla.org/projects/[ PROJECT SLUG ] 

- In this issue: https://github.com/hackforla/website/issues/4031
It currently says the following: 
> Work for LA Page: https://www.hackforla.org/projects/work-for-la.md

It should say:
`Work for LA Page: https://www.hackforla.org/projects/work-for-la`

**Comment by chriskong217** [comment]
- On this issue: https://github.com/hackforla/website/issues/4012
The link to the project leads to the 311 data page, which is a different project. 
This is what the issue currently displays:

> Resources/Instructions
> https://www.hackforla.org/projects/311-data

The link should display the correct Home Unite Us link.

**Comment by ExperimentsInHonesty** [comment]
- One of the issues was stuck because someone sat on it for a month and a half.  I have unassigned them and put it back in the prioritized backlog.  #3998

- Another issue was merged, but the PR had the wrong Issue number as fixed (they used the same number as the PR). #4012

**Comment by t-will-gillis** [comment]
It appears that all 'dependency' issues tracked by this issue have been closed and as a result this issue was moved to "New Issue Approval" (?) and the `Dependency` label was removed. 

Per the last "Action Item", this issue should be moved to "In progress" and then be closed.
